src/commands/generate/imports.ts

```diff
@@ -129,7 +131,7 @@ export function sortImports(imports: ImportDeclarationStructure[]): ImportDeclar
         }
         if (isNamedImport) {
             const curImport = namedImports.get(i.moduleSpecifier);
-            if (curImport != null) {
+            if (curImport != null && Array.isArray(i.namedImports)) {
                 const newImports = i.namedImports!.filter(namedImport => {
```

Comment:
nit: The `!` is likely superfluous now with your `&& Array.isArray(i.namedImports)` check.

Reply:
Nice. Good call.